doc/programming_guide/reference/i2s/i2s_common.rst

```diff
@@ -7,9 +7,9 @@
 ===============
 
 The macro I2S_DATA_WIDTH may be set as a compile flag (e.g. 
--DI2S_DATA_WIDTH=16) to alter the number of bits per word for both the I2S 
+-DI2S_DATA_WIDTH=16) to alter the number of bits per word for both the |I2S| 
 Master and I2S Slave components; this defaults to 32 bits per word. This 
```

Comment:
One more `I2S` that needs changed to `|I2S|`.

Reply:
Good spot, done!